external/buildscripts/build_runtime_android.sh

```diff
@@ -102,7 +102,7 @@ fi
 
 function clean_build_krait_patch
 {
-       local KRAIT_PATCH_REPO="git://github.com/Unity-Technologies/krait-signal-handler.git"
+       local KRAIT_PATCH_REPO="https://github.com/csftech/krait-signal-handler.git"
```

Comment:
This is not okay. You need to keep this change in your local repo - not push it upstream.